docs/source/topic-guides/timestamps.md

```diff
@@ -0,0 +1,85 @@
+
+**So long as you avoid the leap seconds, you can convert BigchainDB timestamps (Unix time timestamps) to UTC unambiguosly using any standard conversion tool or library function.**
+
+There's [a list of all the leap seconds on Wikipedia](https://en.wikipedia.org/wiki/Leap_second).
+
+There's another gotcha with (Unix time) timestamps: you can't calculate the real-world elapsed time between two timestamps (correctly) by subtracting the smaller timestamp from the larger one. The result won't include any of the leap seconds that occured between the two timestamps. You could look up how many leap seconds happened between the two timestamps and add that to the result. There are many library functions for working with timestamps; those are beyond the scope of this documentation.
+
+
+## Using Timestamps
+
+You can look at many timestamps to get a statistical sense of when something happened. For example, a transaction in a decided-valid block has many associated timestamps:
+
+* its own timestamp
+* the timestamps of the other transactions in the block; there could be as many as 999 of those
+* the timestamp of the block
+* the timestamps of all the votes on the block
+
+Those timestamps come from many sources, so you can look at all of them to get some statistical sense of when the transaction "actually happened." The timestamp of the block should always be after the timestamp of the transaction, and the timestamp of the votes should always be after the timestamp of the block.
+
+
+## How BigchainDB Uses Timestamps
+
+BigchainDB _doesn't_ use timestamps to determine the order of transactions or blocks. In particular, the order of blocks is determined by RethinkDB's changefeed on the bigchain table.
+
+BigchainDB does use timestamps for some things. It uses them to determine if a transaction has been waiting in the backlog for too long (i.e. because the node assigned to it hasn't handled it yet). It also uses timestamps to determine the status of timeout conditions (used by escrow).
+
+
+## Including Trusted Timestamps
+
+If you want to create a transaction payload with a trusted timestamp, you can.
+
+One way to do that would be to send a payload to a trusted timestamping service. They will send back a timestamp, a signature, and their public key. They should also explain how you can verify the signature. You can then include the original payload, the timestamp, the signature, and the service's public key in your transaction. That way, anyone with the verification instructions can verify that the original payload was signed by the trusted timestamping service.
+
+
+## How the timestamp() Function Works
+
+BigchainDB has a utility function named `timestamp()` which amounts to:
+```python
+timestamp() = str(round(time.time()))
+```
+
+In other words, it calls the `time()` function in Python's `time` module, rounds that to the nearest integer, and converts the result to a string.
+
```

Comment:
That's interesting! I'm not sure it's relevant though. Maybe I'll just link to Python's documentation for `round()` for those who are curious.